packages/server/src/events.ts

```diff
@@ -36,6 +38,12 @@ export class AgentRunEvent extends WorkflowEvent<{
   data: AgentRunEventData;
 }> {}
 
+export class ReportEvent extends WorkflowEvent<object> {}
+
+export class StopEvent extends StopEventBase<
+  AsyncGenerator<ChatResponseChunk>
+> {}
```

Comment:
not sure why we have these events here - llamaindexserver should only have events that are rendered in the frontend

Reply:
this means we should move the deepresearch events here (as they are rendered in this package)